tests/unit/test_writer_v1.py

```diff
@@ -44,11 +45,15 @@ def _make_one(self, *args, **kw):
         return self._get_target_class()(*args, **kw)
 
     def test_ctor_defaults(self):
+        from google.cloud.bigquery_storage_v1.writer import _clear_proto3_fields
+
         mock_client = self._make_mock_client()
         stream = self._make_one(mock_client, REQUEST_TEMPLATE)
 
         assert stream._client is mock_client
-        assert stream._inital_request_template is REQUEST_TEMPLATE
+        assert stream._initial_request_template == _clear_proto3_fields(
```

Comment:
Should there be a separate test that actually validates changes to the request template?  This simply asserts that the the function was called.

Reply:
Indeed, I just added a unit test for this function